release/models/wifi/openconfig-wifi-mac.yang

```diff
@@ -208,8 +229,19 @@ module openconfig-wifi-mac {
           description
             "WPA2-Enterprise with 802.1X authentication.";
         }
+        enum ENHANCED-OPEN {
+          description
+            "Open authentication with Opportunistic Wireless Encryption.";
+        }
+        enum WPA3_SAE {
+          description
+            "WPA3-SAE using Simultaneous Authentication of Equals (SAE).";
+        }
+        enum WPA3_ENTERPRISE {
```

Comment:
Is there an option for 192 bit security mode with WPA3_ENTERPRISE?

Reply:
Added.